core/error.go

```diff
@@ -32,4 +32,7 @@ var (
 	// ErrNonceTooHigh is returned if the nonce of a transaction is higher than the
 	// next one expected based on the local chain.
 	ErrNonceTooHigh = errors.New("nonce too high")
+
+	// ErrNoGenesis is returned when there is no Genesis Block.
+	ErrNoGenesis = errors.New("Genesis not found in chain")
```

Comment:
Thanks for this. Please also make the `Genesis` lowercase if we're touching the code anyway.

Reply:
I mean in the error message, not the variable.

Reply:
I edited. : )
BTW I saw many other error variable which is placed in other files. Does they are moved to error.go file? If alright, I will edit and commit them.